src/PageCharacter/CharacterDisplay/Tabs/TabOverview/index.tsx

```diff
@@ -0,0 +1,154 @@
+          <StatDisplayComponent />
+        </CardLight>
+      </Grid>
+    </Grid >
+    <Box ref={scrollRef}>
+      <EquipmentSection />
+    </Box>
+  </Stack>
+}
+function EquipmentRow({ onClick }: { onClick: () => void }) {
+  const { character: { equippedWeapon } } = useContext(CharacterContext)
+  const { data, } = useContext(DataContext)
+
+  return <Grid container spacing={1} columns={{ xs: 2, sm: 2, md: 3, lg: 6, xl: 6 }}>
+    <Grid item xs={1}>
+      <WeaponCardNano weaponId={equippedWeapon} BGComponent={CardLight} onClick={onClick} />
+    </Grid>
+    {allSlotKeys.map(slotKey =>
+      <Grid item key={slotKey} xs={1} >
+        <ArtifactCardNano artifactId={data.get(input.art[slotKey].id).value} slotKey={slotKey} BGComponent={CardLight} onClick={onClick} />
+      </Grid>)}
+  </Grid>
+}
+/* Image card with star and name and level */
+function CharacterProfileCard() {
+  const { characterSheet, character: { key: characterKey, team } } = useContext(CharacterContext)
+  const { data, } = useContext(DataContext)
+  const characterDispatch = useCharacterReducer(characterKey)
+  const navigate = useNavigate()
+  const charEle = data.get(input.charEle).value as ElementKey
+  const weaponTypeKey = characterSheet.weaponTypeKey
+  const level = data.get(input.lvl).value
+  const ascension = data.get(input.asc).value
+  const constellation = data.get(input.constellation).value
+  const tlvl = {
+    auto: data.get(input.total.auto).value,
+    skill: data.get(input.total.skill).value,
+    burst: data.get(input.total.burst).value,
+  }
+  const tBoost = {
+    auto: data.get(input.bonus.auto).value,
+    skill: data.get(input.bonus.skill).value,
+    burst: data.get(input.bonus.burst).value,
+  }
+  const [{ favorite }, setCharMeta] = useDBState(`charMeta_${characterKey}`, initCharMeta)
+  return <CardLight sx={{ height: "100%" }} >
+    <Box sx={{ position: "relative" }}>
+      <Box sx={{ position: "absolute", width: "100%", height: "100%" }}>
+        <Typography variant="h6" sx={{ position: "absolute", left: "50%", bottom: 0, transform: "translate(-50%, -50%)", opacity: 0.75 }}>
+          <StarsDisplay stars={characterSheet.rarity} colored />
+        </Typography>
+        <Chip color={charEle} sx={{ height: "auto", position: "absolute", left: "50%", bottom: "5%", transform: "translate(-50%, -50%)", opacity: 0.85, maxWidth: "100%" }}
+          label={<Typography variant="h6" sx={{ display: "flex", gap: 1, alignItems: "center", maxWidth: "100%" }} >
+            {uncoloredEleIcons[charEle]}
+            <Box sx={{ flexGrow: 1, whiteSpace: "normal", textAlign: "center" }} component="span">{characterSheet.name}</Box>
+            <ImgIcon sx={{ pr: 0.5 }} src={Assets.weaponTypes?.[weaponTypeKey]} />
+          </Typography>} />
+
+        <Box sx={{ position: "absolute", left: 0, top: 0 }}>
+          <IconButton sx={{ p: 1 }} color="error" onClick={() => setCharMeta({ favorite: !favorite })}>
```

Comment:
Should we remove the color? Or add the color to the other heart locations?

Reply:
I tried to make others red, but it popped out too much. In this instance, it has to be red because the background is white.